packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -302,6 +302,15 @@ contract Attestations is
     emit Withdrawal(msg.sender, token, value);
   }
 
+  function withdrawToSigner(address token) external {
+    address issuer = getAccounts().attestationSignerToAccount(msg.sender);
+    uint256 value = pendingWithdrawals[token][issuer];
+    require(value > 0, "value was negative/zero");
+    pendingWithdrawals[token][issuer] = 0;
+    require(IERC20(token).transfer(msg.sender, value), "token transfer failed");
+    emit Withdrawal(msg.sender, token, value);
+  }
+
```

Comment:
I think we should add logic to the `withdraw` function to allow the attestation signer to withdraw instead of adding a new function. But I can also see the argument for doing with with a new function instead